core/src/main/java/org/axonframework/config/EventHandlingConfiguration.java

```diff
@@ -89,24 +115,6 @@
     public EventHandlingConfiguration() {
     }
 
-    private SubscribingEventProcessor defaultEventProcessor(Configuration conf, String name, List<?> eh) {
-        return subscribingEventProcessor(conf, name, eh, Configuration::eventBus);
-    }
-
-    private SubscribingEventProcessor subscribingEventProcessor(Configuration conf, String name, List<?> eh,
-                                                                Function<Configuration, SubscribableMessageSource<? extends EventMessage<?>>> messageSource) {
-        return new SubscribingEventProcessor(name,
-                                             new SimpleEventHandlerInvoker(
-                                                     eh,
-                                                     conf.parameterResolverFactory(),
-                                                     getListenerInvocationErrorHandler(conf, name)
-                                             ),
-                                             messageSource.apply(conf),
-                                             DirectEventProcessingStrategy.INSTANCE,
-                                             getErrorHandler(conf, name),
-                                             getMessageMonitor(conf, SubscribingEventProcessor.class, name));
-    }
-
     /**
      * Returns the list of Message Handler Interceptors registered for the given {@code processorName}.
      *
```

Comment:
Shouldn't the `interceptorsFor(Configuration, String)` function be copied over to the `EventProcessorRegistry` as well?

Reply:
Will introduce `interceptorsFor(String)` in `EventProcessorRegistry`.